README.md

```diff
@@ -115,7 +115,8 @@ jobs:
 ## Authorization
 
 This action installs the Cloud SDK (`gcloud`). To configure its authentication
-to Google Cloud, use the [google-github-actions/auth][auth] action. You can
+to Google Cloud, use the [google-github-actions/auth][auth] action. This action sets both
```

Comment:
setup-gcloud [_does not_ set ADC](https://github.com/google-github-actions/setup-cloud-sdk/blob/e1f05c656f9776325bd4ec36b6aec0f155aa140c/src/index.ts#L225), only `auth` does that.

Reply:
Thanks!